src/templates/components/hours/index.tsx

```diff
@@ -55,14 +61,26 @@ export const Hours = ({
         )
       case 'clinical':
         return (
-          <h2 className="vads-u-margin-top--2p5 vads-u-margin-bottom--1">
-            Clinical hours
-          </h2>
+          <>
+            <h3
+              className="vads-u-margin-top--2p5 vads-u-margin-bottom--1"
+              id="hours-heading"
+            >
+              Facility hours
+            </h3>
+            <p>
+              Hours may vary for different services. Select a service on this
+              page to check the hours.
+            </p>
+          </>
```

Comment:
This is the main change. From my search, I didn't see anything else that used this `'clinical'` `headerType`, so it should be scoped to just facilities.